server/src/main/java/org/apache/cassandra/sidecar/server/MainModule.java

```diff
@@ -380,6 +382,9 @@ public Router vertxRouter(Vertx vertx,
         router.get(ApiEndpointsV1.GOSSIP_INFO_ROUTE)
               .handler(gossipInfoHandler);
 
+        router.get(ApiEndpointsV1.GOSSIP_STATUS_ROUTE)
```

Comment:
Do we really need a new endpoint having GOSSIP_INFO_ROUTE already? I see that the GossipInfoResponse already has a deprecated STATUS field. Maybe finding out why/where it was deprecated, and hijacking it to use this `storageOperations::isGossipRunning` can save us from having to add this new API?

Reply:
As we discussed offline, we cannot infer gossip running status from STATUS (or STATUS_WITH_PORT) fields of gossip info. Hence we cannot use gossip/filter=status for gossip running status, as that filter can be mistaken for STATUS field of gossip info. Will continue with gossip/status endpoint (Other option could be gossip/runningStatus, but sounds too verbose)